.github/pull_request_template.md

```
@@ -0,0 +1,152 @@
+
+### 🎯 Use Cases
+<!-- Describe specific use cases this addresses -->
+
+### 📝 Notes
+<!-- Any additional notes or considerations -->
+
+---
+
+## 📞 Reviewer Notes
+
+### 👥 For Reviewers
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)